src/algebra/group/to_additive.lean

```diff
@@ -104,6 +140,23 @@ meta def reorder_attr : user_attribute (name_map $ list ℕ) (list ℕ) :=
 
 end extra_attributes
 
+/--
+Find the first argument of `nm` that has a multiplicative type-class on it.
+Returns 1 if there are no types with a multiplicative class as arguments.
```

Comment:
Why is 1 a useful thing to return when nothing is found?

Reply:
Two reasons:
(1) `1` was already the default value we used for the position of the "relevant argument", which works for 98+% of declarations
(2) The first argument is almost always a type,`(_ : Type*)`, and if there are no type arguments that have a multiplicative structure, the best we can do is pick one at random. If you think it's better we could pick the first argument that has as type `Sort _`.